websrc/.vscode/launch.json

```diff
@@ -63,7 +63,17 @@
 			"sourceMaps": true,
 			"port": 9229,
 			"protocol": "auto",
-			"outFiles": [ "${workspaceRoot}/myOutFiles/**/*.js",  "${workspaceRoot}/node_modules/**/*.js" ]
+			"outFiles": [  
+				"${workspaceRoot}/node_modules/**/*.js",
+				"${worskpaceRoot}/../packages/aardvark-shared/**/*.js",
+				"${worskpaceRoot}/../packages/aardvark-shared/**/*.js.map",
+				"${worskpaceRoot}/../packages/aardvark-shared/**/*.ts",
+				"${worskpaceRoot}/../packages/aardvark-shared/**/*.js.map",
+				"${worskpaceRoot}/../packages/aardvark-react/**/*.js",
+				"${worskpaceRoot}/../packages/aardvark-react/**/*.js.map",
+				"${worskpaceRoot}/../packages/aardvark-react/**/*.ts",
+				"${worskpaceRoot}/../packages/aardvark-react/**/*.js.map",
+			 ]
```

Comment:
Huh. Does this fix source maps in node even without local builds of the packages?

Nice that it supports relative paths.

Reply:
this got the source maps working for me, but I also have a watch compiling the aardvark-shared and aardvark-react in the packages dir so that might have been redundant but got the source maps working.